package.xml

```diff
@@ -14,9 +14,11 @@
   <url type="repository">https://github.com/ros/rosconsole_bridge</url>
 
   <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>cpp_common</build_depend>
   <build_depend>libconsole-bridge-dev</build_depend>
   <build_depend version_gte="1.11.5">rosconsole</build_depend>
 
+  <run_depend>cpp_common</run_depend>
```

Comment:
hi @dirk-thomas, thanks for adding the `run_depend` dependency! just curious, why is `cpp_common` considered a runtime dependency? I thought we only need this for the header, so it's only build/compile time.